src/middlewared/middlewared/plugins/pool.py

```diff
@@ -3509,6 +3509,22 @@ async def do_update(self, id, data):
         if data.get('user_properties_update'):
             props.update(await self.get_create_update_user_props(data['user_properties_update'], True))
 
+        if 'acltype' in props:
```

Comment:
Just a nit but instead of calling `props['acltype'].get('value')` in 3 separate branches you could do

`if 'acltype' in props and (acltype_value := props['acltype'].get('value')):
    if acltype_value == 'nfsv4':
        ....